src/javascript/_common/__tests__/tests_common.js

```diff
@@ -5,7 +5,8 @@ const websocket   = require('ws');
 const Client      = require('../../app/base/client');
 const Language    = require('../language');
 const Url         = require('../url');
-
+// ignore svgs in tests
+require.extensions['.svg'] = () => '<svg></svg>';
```

Comment:
why?

Reply:
Not ignoring SVGs would result into `>> SyntaxError: /src/images/app_2/settings/img-theme-dark.svg: Namespace tags are not supported. ReactJSX is not XML.` Which is in part because of `svg-inliner` not working properly. So, we can ignore all svg files from injecting into tests to not to see this message & speed things up a bit.

Reply:
can you add that as comment in the code?